translations/fr/try_ruby_120.md

```diff
@@ -0,0 +1,21 @@
+---
+lang:   EN
+title:  Faire la queue
+answer: ^\[(\d+)(,\s*\d+){3,}\]$
+ok:     Excellent
+error:  Essaye d'ajouter un quatrième nombre à la liste
+---
+
+Les listes (arrays) stockent des choses __dans un ordre indéfini__.
+
+C'est un peu comme faire la queue pour acheter une baguette. Tu es
+derrière quelqu'un et ce sera impensable de les pousser pour passer
+devant, non? Et le mec derrière toi, tu gardes un oeil sur lui, juste au
+cas où ?
```

Comment:
Here I'd add a `, n'est-ce pas ?` or a `, non ?` like the previous sentence, since it isn't actually a question that asks for an answer.

Reply:
`non?` is a bit more colloquial and less formal than "n'est-ce pas?" so I'll keep it. Trying to keep the register low brow.